src/processes/follow-topic-process.js

```diff
@@ -92,9 +136,35 @@ const sendMessageAsAnonymous = async (serverClient, communityMessageFormat, data
   }
   // }
 };
+
+const sendSignedMessage = async ({
+  serverChannel,
+  targetUserId,
+  senderUsername,
+  communityMessageFormatUserId,
+  text
+}) => {
+  const message = await serverChannel?.sendMessage(
+    {
+      text,
+      user_id: communityMessageFormatUserId,
+      is_auto_message: true
+    },
+    {skip_push: true}
+  );
+
+  sendPushNotification({
+    serverChannel,
+    serverMessage: message,
+    targetUserId,
+    senderUsername,
+    text
+  });
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `sendSignedMessage` does not handle any potential errors that might occur when calling `sendMessage` and `sendPushNotification`. It's recommended to add error handling to ensure the application can gracefully handle any issues that arise.
```diff
+  try {
      const message = await serverChannel?.sendMessage(
        {
          text,
          user_id: communityMessageFormatUserId,
          is_auto_message: true
        },
        {skip_push: true}
      );
  
      sendPushNotification({
        serverChannel,
        serverMessage: message,
        targetUserId,
        senderUsername,
        text
      });
+  } catch (error) {
+    console.error('Error sending signed message:', error);
+    throw error;
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->